src/schemas/receiver.test.js

```diff
@@ -92,7 +102,7 @@ describe('Receiver Validation', () => {
 
     const { error } = validate(receiver, receipt)
     expect(error).toBeDefined()
-    expect(error.message).toBe('"receiver.authorisationNumbers" is required')
+    expect(error.message).toBe('Site authorisation number is required')
```

Comment:
let's keep the old error message, it's formatted like the original error messages and it's clearer for developers to address